plugins/modules/packaging/os/pacman.py

```diff
@@ -44,6 +44,11 @@
         default: no
         type: bool
 
+    bin:
+        description:
+            - Name of pacman binary or AUR helper that shares the same interface as pacman.
```

Comment:
What do you think about adding a warning here that using AUR helpers are not recommended and may have weird behaviours.